misc/helper-scripts/taste-create-data-view

```diff
@@ -97,7 +117,60 @@ fi
 
 # cp $(taste-config --prefix)/share/taste-types/taste-types.asn .
 
-taste-update-data-view
+# If ROS messages will be generated
+if [ $GEN_ROS_MSGS ]
+then
+    # Generate TASTE extended types
+    echo 'TASTE-ExtendedTypes DEFINITIONS ::=
+    BEGIN
+
+    IMPORTS
+    T-Int32, T-UInt32, T-Int8, T-UInt8, T-Boolean  FROM TASTE-BasicTypes;
+
+    -- Set of TASTE predefined basic types
+
+    T-Double ::= REAL (-1.7976931348623157e+308..1.7976931348623157E308)
```

Comment:
For symmetry, add + after E at the upper end of the range.

Reply:
Done